files/packs.json

```diff
@@ -586,7 +586,7 @@
             "3;All PvP"
         ],
         "file": "FishFault.zip",
-        "url": "https://github.com/Sean2739/Fishfault/releases/download/3.0.2/FishFault.zip",
```

Comment:
you uploaded fishfault so you no longer need the url
also yes please rehash, here on linux we can just pipe a file to `md5sum` (eg `cat abc | md5sum` or `curl -L https://abc | md5sum`), you probably can do something similar in powershell